criu/files-reg.c

```diff
@@ -231,8 +297,11 @@ static int copy_chunk_to_file(int img, int fd, off_t off, size_t len)
 
 		if (opts.stream)
 			ret = splice(img, NULL, fd, NULL, len, SPLICE_F_MOVE);
-		else
+		else {
```

Comment:
@featherchen Would you be able to add braces for both branches of the conditional statement? We tend to follow the [Kernel Coding Style](https://www.kernel.org/doc/html/latest/process/coding-style.html?highlight=coding+style#placing-braces-and-spaces) in CRIU.

Reply:
Sure, I have add them.
btw, is there any  auto way to do this (similar to make indent)

Reply:
> btw, is there any auto way to do this (similar to make indent)

I believe no, there is no automatic way to enforce this. The clang-format 15 have only "use brackets everywhere" option which is not what LKCS says.